From 1e47a5ad4e0ba5826f50a790a71a387a1d950377 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Fri, 17 Dec 2021 17:16:14 -0500
Subject: [PATCH] Reduce the size of udfps enroll progress bar.

Some devices have their udfps pretty low, and the progress bar
would make the icon at the wrong place

Change-Id: I1609ad9ca316293dcaaf07f7e681d11aadfcd29c
---
 packages/SystemUI/res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/res/values/config.xml b/packages/SystemUI/res/values/config.xml
index 54c641de0bc6..3d1509d3b1dd 100644
--- a/packages/SystemUI/res/values/config.xml
+++ b/packages/SystemUI/res/values/config.xml
@@ -577,7 +577,7 @@
 
     <!-- The radius of the enrollment progress bar, in dp -->
     <integer name="config_udfpsEnrollProgressBar" translatable="false">
-        280
+        70
     </integer>
 
     <!-- The time (in ms) needed to trigger the lock icon view's long-press affordance -->
-- 
2.34.1

